src/test/java/net/sf/jabref/event/AddEntryEventTest.java

```diff
@@ -0,0 +1,33 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.IdGenerator;
+
+import com.google.common.eventbus.EventBus;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AddEntryEventTest {
+
+    private EventBus eventBus;
+    private TestEventListener testEventListener;
+
+    @Before
+    public void setUp() {
+        eventBus = new EventBus();
+        testEventListener = new TestEventListener();
+        eventBus.register(testEventListener);
+    }
+
+    @Test
+    public void testEventReceivement() {
+        BibEntry shouldBeBibEntry = new BibEntry();
+        shouldBeBibEntry.setId(IdGenerator.next());
+        AddEntryEvent aocee = new AddEntryEvent(shouldBeBibEntry);
+        eventBus.post(aocee);
```

Comment:
You are actually retesting the EventBus class which is not required. Only test our code.